.idea/runConfigurations/CertificateItemsControllerUnitTest.xml

```diff
@@ -0,0 +1,20 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Should these files be commited? Isn't this specific to Intellij?

Reply:
Yes, these are specific to IntelliJ. I committed them because otherwise these tests can mysteriously fail when run in IntelliJ because the download of embedded Mongo fails to get through CH proxies.

Reply:
Can the IntelliJ specific files be removed please?

Reply:
Ok, will do.